format-flowed/src/lib.rs

```diff
@@ -52,7 +52,7 @@ fn format_line_flowed(line: &str, prefix: &str) -> String {
             after_space = false;
         }
     }
-    result + &buffer
+    (result + &buffer).trim_end().to_string()
```

Comment:
wondering if the `trim_end()` here and in `format_flowed()` below is really needed - there are languages as markdown where [trailing spaces are significant](https://daringfireball.net/projects/markdown/syntax#p).

maybe not a big deal, just saying, this pr seems not to make that worse as trimming was done before already.